kilkari/src/main/java/org/motechproject/nms/kilkari/domain/Subscriber.java

```diff
@@ -16,16 +18,17 @@
     @Unique
     private String callingNumber;
 
-    //todo: should this be a Set<> instead of a List<>?
+    //making this a bi-directional relationship until MOTECH-1638 is fixed
```

Comment:
Frank's checkin had the todo, mine removes it. I think he skirted the checkstyle warning by using lowercase todo rather than TODO :)

I'll change to lazy loading.